spec/factories/doctors_notes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :doctors_note do
+    passenger_id 1
+    expiration_date "2017-07-27"
+    overriden_by 1
```

Comment:
Well, I was thinking of just leaving the factory with one line for the passenger association since that is the only one that would be needed. Therefore, everything else would be filled out in the tests as needed.